src/SampleLibraryUI/Examples/LocalizedCustomNodeModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SampleLibraryUI.Examples
+{
+    /// <summary>
+    /// This example node uses .net .resx files and generated sattelite assemblies to perform runtime lookup of localized content
+    /// depending on the culture of the system Dynamo is running on.
+    /// Read more: https://docs.microsoft.com/en-us/dotnet/framework/resources/creating-resource-files-for-desktop-apps
+    /// You can use the -l "es-ES" flag when starting DynamoSandbox.exe to replace the English strings with Spanish ones.
+    /// </summary>
+    [NodeName("LocalizedNode")]
+    [NodeDescription("CustomNodeModelDescription", typeof(Properties.Resources))]
+    [OutPortNames("string")]
+    [OutPortTypes("string")]
+    [IsDesignScriptCompatible]
+    public class LocalizedCustomNodeModel : NodeModel
```

Comment:
Any specific reason why you chose a NodeModel node vs. a zero touch node?

Reply:
I choose a nodeModel so I could show an example of using the attributes - I think the syntax is harder to get right so worth showing.